demos/system_reference-ST.rst

```diff
@@ -106,10 +106,10 @@ microSD card inserted on your HOST PC using command
 .. code-block:: console
 
    cd tmp-glibc/deploy/images/stm32mp15-disco-oss/
-   # Eject potential media mounted from the sdcardreplace <device> by mmcblk<X> (X = 0,1..) or
+   # Umount potential media mounted from the sdcard. Replace <device> by mmcblk<X> (X = 0,1..) or
```

Comment:
"Umount" or "Unmount"? Since the actual command is "eject", I think "Unmount" is more accurate.